amxmodx/newmenus.cpp

```diff
@@ -1076,6 +1076,112 @@ static cell AMX_NATIVE_CALL menu_setprop(AMX *amx, cell *params)
 	return 1;
 }
 
+// Gets a menu property
+// native menu_getprop(menu, prop, ...);
+static cell AMX_NATIVE_CALL menu_getprop(AMX *amx, cell *params)
+{
+	enum args { arg_numargs, arg_menu, arg_prop, arg_buffer, arg_bufferlen };
+
+	GETMENU(params[arg_menu]);
+
+	int paramsNum = params[arg_numargs] / sizeof(cell);
+
+	switch(params[arg_prop])
```

Comment:
Missing space between `switch` and `(params[arg_prop])`.

Reply:
Done.